packages/babel-check-binary-expressions/src/index.js

```diff
@@ -0,0 +1,8 @@
+export default function(left, right) {
+  // eslint-disable-next-line no-undef
+  if (left instanceof BigInt && right instanceof BigInt) {
```

Comment:
yes, that makes sense. if I use `global.BigInt`, will it work in the browser and Node?